modules/op-specifying-manual-approval-task.adoc

```diff
@@ -0,0 +1,59 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-specifying-manual-approval-task_{context}"]
+= Specifying a manual approval task
+
+You can specify a manual approval task in your pipeline. When the execution of a pipeline run reaches this task, the pipeline run stops and awaits approval from one or several users.
+
+.Prerequisites
+
+* You enabled the manual approver gate controller.
+* You created a YAML specification of a pipeline.
+
+.Procedure
+
+* Specify an `ApprovalTask` in the pipeline, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: example-manual-approval-pipeline
+spec:
+  tasks:
+# ...
+  - name: example-manual-approval-task
+    taskRef:
+      apiVersion: openshift-pipelines.org/v1alpha1
+      kind: ApprovalTask
+    params:
+    - name: approvers
+      value:
+      - user1
+      - user2
+      - user3
+    - name: description
+      value: Example manual approval task - please approve or reject
+    - name: numberOfApprovalsRequired
+      value: '2'
+    - name: timeout
+      value: '60m'
+# ...
+----
++
+.Parameters for a manual approval task
```

Comment:
This table feels out of place here and should be in a reference module.

Reply:
It is consistent with many other similar places in this document. Example: https://docs.openshift.com/pipelines/1.15/create/remote-pipelines-tasks-resolvers.html#resolver-bundles-specify_remote-pipelines-tasks-resolvers

 This same thing could be done using callouts but I think the table reads better. And one needs all the parameters exactly when specifying the task so I don;t think a separate module would meet the self sufficiency requirement.

Reply:
OK. Thanks for clarifying.